src/Plugin_Command.php

```
@@ -1152,26 +1159,35 @@ protected function get_wporg_data( $plugin_name ) {
 	}
 
+	/**
+	 * Formats a wp.org publish date as a `Y-m-d` string in the site's configured timezone.
+	 *
+	 * @param int|null $pub_date Unix timestamp, or null if it could not be parsed.
+	 *
+	 * @return string
+	 */
+	private function format_wporg_last_updated( $pub_date ) {
+		if ( function_exists( 'wp_date' ) ) {
+			return wp_date( 'Y-m-d', $pub_date );
```

PHPStan (2.2.8)

[error] Method Plugin_Command::format_wporg_last_updated() should return string but returns string|false.


(return.type)